plugins/org.wso2.developerstudio.eclipse.esb.dashboard.templates/Samples/ContentBasedRoutingTemplate/src/main/synapse-config/proxy-services/ArithmaticOperationService.xml

```diff
@@ -3,33 +3,55 @@
     <target>
         <inSequence>
             <log description="Log Request Payload" level="full"/>
-            <switch description="Retrieve the first element name and match in the case. Otherwise, send back custom payload." source="local-name(//soapenv:Body/*[1])" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
-                <case regex="AddInteger">
-                    <header name="Action" scope="default" value="http://tempuri.org/SOAP.Demo.AddInteger"/>
-                    <send description="Send request to the endpoint">
+            <switch description="Route based on the Operation" source="//ArithmaticOperation/Operation" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
+                <case regex="Add">
+                    <payloadFactory description="Build backend payload" media-type="xml">
```

Comment:
Shall we say "construct payload for addition operation"

Reply:
Fixed.